### PR TITLE
add example to deploy faasd

### LINKED ISSUE #440: Add an example for faasd
For the faasd project, we tend to use Multipass which now has support for preloaded examples and cloud-init.

I'd like to see someone from the Lima or OpenFaaS community contribute an example for faasd:

https://github.com/lima-vm/lima/tree/master/examples

I'd probably suggest using Ubuntu as a base image.

Here's a 3 line installation script, so I think it'd look just like one of the existing examples, plus this in the `provision:` section of the YAML file.

https://github.com/openfaas/faasd#deploy-faasd

### PR DESCRIPTION
**Pull request by jsiebens**
Add an example to deploy faasd

fixes #440

### CODE REVIEW

File: examples/faasd.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+#  version: 0.21.0
+#  sha:     539f0a2c946ee6096ef2b6ab26f81295eed49d54
+#
+#
+# Provider
+#  name:          faasd
+#  orchestration: containerd
+#  version:       0.14.3
+#  sha:           ea62c1b12dd1ae1de794e6dd260351cfbd1a6759
+# Your faas-cli version (0.13.13) may be out of date. Version: 0.13.15 is now available on GitHub.
```
Comment (afbjorklund) [inline examples/faasd.yaml:31]: I think you want to remove the banner and the version, so that the yaml stays relevant for longer ?

Comment (jsiebens) [reply examples/faasd.yaml]: yes, you're right, I've changed it with another example command to make it more future proof and less verbose

File: examples/faasd.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Deploy faasd (which installs a bundled containerd).
+#
+# It can be accessed from the host by authenticating with faas-cli;
+# the ports are already forwarded automatically by lima:
+#
+# $ export OPENFAAS_URL=http://localhost:8080
+# $ limactl shell faasd sudo cat /var/lib/faasd/secrets/basic-auth-password | faas-cli login -u admin --password-stdin
```
Comment (AkihiroSuda) [inline examples/faasd.yaml:7]: nit: Could you add how to install `faas-cli`  on the host?

Comment (jsiebens) [reply examples/faasd.yaml:11]: done

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**